app/src/test/java/com/mobiledevelopmentfoodapp/ExampleUnitTest.kt

```diff
@@ -4,11 +4,7 @@ import org.junit.Test
 
 import org.junit.Assert.*
 
-/**
- * Example local unit test, which will execute on the development machine (host).
- *
- * See [testing documentation](http://d.android.com/tools/testing).
- */
+/** Example local unit test, which will execute on the development machine (host). */
```

Comment:
Simplified this comment according to the following Kotlin coding convention. https://kotlinlang.org/docs/coding-conventions.html#documentation-comments